ethergo/submitter/submitter.go

```
@@ -421,12 +421,47 @@
 		//nolint: wrapcheck
 		return parentTransactor.Signer(address, transaction)
 	}
+	// if dynamic gas estimation is not enabled, use cfg var gas_estimate as a gas limit default and do not run a pre-flight simulation
+	// since we do not need it to determine proper gas units
+	if !t.config.GetDynamicGasEstimate(int(chainID.Uint64())) {
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion uint64 -> int

(gosec)